### DISCUSSION TITLE
Implement Geni messaging integration?

### DISCUSSION THREAD

**Discussion by Sternbach-Software** [Sternbach-Software/GeniForAndroid#4] [General]
This is a place to discuss the necessity/convenience of supporting Geni messaging. Understand that this is not a trivial feature, and adding integration will increase the workload for maintaining the app, and will take away time from working on core features. I haven't looked if they have an API for this, but if there isn't one yet and there is enough demand, they may make one.

**Comment by carsrac** [comment]
As far as I know sending messages using the API is not possible, but this question should be asked on Geni platform and in the correct discussion group.
